### ISSUE TITLE
Why is the Performance on the BUSI Unknown Domain So Poor?

### ISSUE THREAD

**Issue by ambitionxuyi-lgtm** [MQinghe/SynFoC#8]
[18:19:44.792] epoch 60 : loss : 0.000000
	val_base_dice: 0.515471, 
	val_base_dc: 0.515368, 	val_base_jc: 0.453788, 
	val_base_hd: 50.908523, 	val_base_asd: 34.988908, 
[18:19:44.792] val_base_best_dice: 0.588033 at 22500 iter, val_best_avg_dice: 0.588033 at 22500 iter
[18:19:44.792] test sam model
[18:19:46.947] domain1 epoch 60 : loss : 0.000000
	val_base_dice: 0.829903, 
	val_base_dc: 0.829852, 	val_base_jc: 0.751010, 
	val_base_hd: 21.384522, 	val_base_asd: 6.820210, 
[18:19:47.955] domain2 epoch 60 : loss : 0.000000
	val_base_dice: 0.384196, 
	val_base_dc: 0.384108, 	val_base_jc: 0.299637, 
	val_base_hd: 62.661493, 	val_base_asd: 25.252134, 
[18:19:47.956] epoch 60 : loss : 0.000000
	val_base_dice: 0.607050, 
	val_base_dc: 0.606980, 	val_base_jc: 0.525324, 
	val_base_hd: 42.023007, 	val_base_asd: 16.036172, 
[18:19:47.956] stu_val_base_best_dice: 0.705726 at 5000 iter, val_best_avg_dice: 0.705726 at 5000 iter

**Comment by ambitionxuyi-lgtm** [comment]
How Are the Hyperparameters in the Training Script Set?

**Comment by Hzy3290** [comment]
From my perspective, the BUSI dataset is not fit for MiDSS setting, because diferent domain should contain the same classes. In the BUSI dataset, two domains are two seperate categories respectively, which means if the labeled samples belong to class A, then the images in unknown domain belong to class B. It is nearly impossible to transfer the knowledge of class A to class B in this setting. That's why the performance on the BUSI  unknown domain is so poor.
